pypackage/{{ cookiecutter.slug }}/src/{{ cookiecutter.package_name }}/cli.py

```diff
@@ -2,12 +2,13 @@
 from importlib.metadata import version
 
 
-def cli(_argv=None):  # pylint:disable=inconsistent-return-statements
+def cli(argv=None):
```

Comment:
I'm not sure why `argv` was named with a leading underscore here, that's not necesary